src/JSInterop/src/Mono.WebAssembly.Interop/Mono.WebAssembly.Interop.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- API Check fails with "Generic method or method in generic class is internal call, PInvoke, or is defined in a COM Import class." -->
+    <EnableApiCheck>false</EnableApiCheck>
+    <IsPackable>false</IsPackable>
```

Comment:
Not bothering about this package for now since it's not used by Razor Components. We can fix this at a later point